src/iiif/ingress_email_consumer.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from ingress.consumer.base import BaseConsumer
+
+from iiif import mailing
+from main import settings
+
+
+class EmailConsumer(BaseConsumer):
+    collection_name = settings.EMAIL_COLLECTION_NAME
+
+    """
+    Whether or not to immediately remove messages once consumption succeeds.
+    If set to False, message.consume_succeeded_at will be set.
+    """
+    remove_message_on_consumed = True
+
+    """
+    Whether or not to set Message.consume_started_at immediately once consumption starts
+    """
+    set_consume_started_at = False
+
+    def consume_raw_data(self, raw_data):
+        record = json.loads(raw_data)
+        mailing.send_email(record['email'], record['subject'], record['body'])
```

Comment:
Ik mis een testcase voor de consumer.

Reply:
Die wordt toch gewoon in tests/test_iiif.py op regel 389 getest?

Reply:
daar test je enkel dat er na het draaien van de consumer er geen `Message` en geen `FailedMessage` is. 

Echter, stel dat de consumer `Message.objects.all().delete()` doet, dan passed de test ook. Ik zou eerder een mini-test toevoegen die daadwerkelijk checkt of de `send_email()` met de juiste params wordt aangeroepene.

Reply:
Ah ja, dat kan. Gezien hij zo minimaal is, en gezien die send_email niet echt getest kan worden vond ik het niet nodig. Maar zal er een testje aan toevoegen.